### ISSUE TITLE
AssertationError: Shortcodes disappear from the 4th post from your feed

### ISSUE THREAD

**Issue by tim54000** [instaloader/instaloader#478]
**Describe the bug**
After several attempts with one account and several unsubscriptions among others, I deduced that the shortcodes are not retrieved at the 4th post of your feed. 
Indeed, by using this account via the following command `instaloader -l <my_account> -p ####### :feed`, only 3 posts are recovered before failing with the error described later.

By using an older account, it seems that the error does not recur.

**To Reproduce**
- (Maybe creating a new account is needed to reproduce the bug)
- Just launch `instaloader -l <your_account> :feed`

**Expected behavior**
The program connects successfully and retrieves 3 posts from your feed before failing with the following error.

**Error messages and tracebacks**
```Session file does not exist yet - Logging in.
Logged in as <my_account>.
Retrieving pictures from your feed...
[  1] <some_account> :feed/2019-12-13_21-08-24_UTC_1.jpg exists :feed/2019-12-13_21-08-24_UTC_2.jpg exists [...] unchanged json 
[  2] <some_account2> :feed/2019-12-13_20-13-13_UTC.jpg exists [...] unchanged json 
[  3] <some_account3> :feed/2019-12-12_11-57-49_UTC.jpg [...] json 
Traceback (most recent call last):
  File "/usr/bin/instaloader", line 11, in <module>
    load_entry_point('instaloader==4.2.10', 'console_scripts', 'instaloader')()
  File "/usr/lib/python3.7/site-packages/instaloader/__main__.py", line 414, in main
    storyitem_filter_str=args.storyitem_filter)
  File "/usr/lib/python3.7/site-packages/instaloader/__main__.py", line 151, in _main
    post_filter=post_filter)
  File "/usr/lib/python3.7/site-packages/instaloader/instaloader.py", line 49, in call
    return func(instaloader, *args, **kwargs)
  File "/usr/lib/python3.7/site-packages/instaloader/instaloader.py", line 706, in download_feed_posts
    for post in self.get_feed_posts():
  File "/usr/lib/python3.7/site-packages/instaloader/instaloader.py", line 675, in get_feed_posts
    yield from (Post(self.context, edge["node"]) for edge in feed["edges"]
  File "/usr/lib/python3.7/site-packages/instaloader/instaloader.py", line 676, in <genexpr>
    if not edge["node"]["__typename"] == "GraphSuggestedUserFeedUnit")
  File "/usr/lib/python3.7/site-packages/instaloader/structures.py", line 66, in __init__
    assert 'shortcode' in node or 'code' in node
AssertionError

```

**Version**
Instaloader: 4.2.10 *(installed via `pip install instaloader`)*
Python: 3.7 & 3.8.1
OS: 5.3.7-2-MANJARO x86_64 GNU/Linux

**Additional context**
The account concerned was created on December 14, 2019.
It is possible that the problem is related to the issue #467. The context being different and the link between the error not obvious, I preferred to create a new issue.

**Comment by tim54000** [comment]
Issue #467 (Duplicate of #449) has been closed and appears to be resolved.
However, the error reported here still occurs with the same account.

**Comment by tim54000** [comment]
Suspected link to exit #503.

**Comment by krummrey** [comment]
Just wanted to say, that I'm having the same issue on multiple occasions on multiple accounts.

**Comment by aandergr** [comment]
We fixed this with dc458a3f3c8f7d174c5c5596508c4f57685a7181 which will be included in the upcoming version, Instaloader 4.4. To already try it out, upgrade Instaloader with:

```
pip3 install --upgrade git+https://github.com/instaloader/instaloader@v4.4-dev
```

**Comment by tim54000** [comment]
Effectively, I can now download more than 3 photos from the feed. Thanks for the correction, I close the issue.
